yt/frontends/boxlib/data_structures.py

```diff
@@ -718,8 +717,6 @@ def _parse_cparams(self):
         for line in (line.split("#")[0].strip() for line in open(self.cparam_filename)):
             if "=" not in line:
                 continue
-            if len(line) == 0:
```

Comment:
this has nothing to do with the PR but I think this check is 100% redundant with the one above so I simply removed it